airtable/backup_airtable.py

```diff
@@ -2,44 +2,18 @@
 import datetime
 import os
 import tempfile
-import urllib
 import zipfile
 
 from boto.s3.connection import S3Connection
 from boto.s3.key import Key
-import requests
+import airtable
```

Comment:
Oh yeah, for cron jobs we'll get an email if the process errors out, but if something fails silently (without an error) then we won't know. Probably don't need to worry about that.

We might want to think about this for stuff like the map though. We want to know when it goes down.